openshift/templates/aries-mediator-service/aries-mediator-service-deploy.yaml

```diff
@@ -7,7 +7,31 @@ metadata:
     tags: node
     iconClass: icon-node
 objects:
-
+  - kind: NetworkPolicy
+    apiVersion: networking.k8s.io/v1
+    metadata:
+      name: ${NAME}${SUFFIX}-allow-ingress
+      labels:
+        name: ${NAME}${SUFFIX}-allow-ingress
+        app: ${APP_NAME}
```

Comment:
All `app` labels should follow the ${APP_NAME}${SUFFIX} naming syntax to be consistent with the rest of the configurations.  Use https://github.com/bcgov/orgbook-configurations/blob/main/openshift/templates/agent/agent-deploy.yaml#L8-L32 as a reference.

Reply:
Change completed